pkg/controller/deployment/sync.go

```diff
@@ -534,22 +534,6 @@ func (dc *DeploymentController) isScalingEvent(d *extensions.Deployment) (bool,
 	if err != nil {
 		return false, err
 	}
-	// If there is no new replica set matching this deployment and the deployment isn't paused
-	// then there is a new rollout that waits to happen
-	if newRS == nil && !d.Spec.Paused {
-		// Update all active replicas sets to the new deployment size. SetReplicasAnnotations makes
-		// sure that we will update only replica sets that don't have the current size of the deployment.
-		maxSurge := deploymentutil.MaxSurge(*d)
-		for _, rs := range controller.FilterActiveReplicaSets(oldRSs) {
-			if updated := deploymentutil.SetReplicasAnnotations(rs, d.Spec.Replicas, d.Spec.Replicas+maxSurge); updated {
```

Comment:
Yes, since we are going to proportionally scale and not rollout, these replica set annotations will be updated by the code in `scale`.